doodle-trace/pom.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.gluonhq.samples</groupId>
+    <artifactId>doodle-trace</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Doodle Trace</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <javafx.version>15-ea+1</javafx.version>
+        <attach.version>4.0.6</attach.version>
+        <mainClassName>com.carlfx.GluonApplication</mainClassName>
+        <javafx-maven-plugin-version>0.0.4</javafx-maven-plugin-version>
```

Comment:
I would opt for consistent naming of properties, by using the dot-separated names. You can also drop the -maven- part, because we are in a maven project after all? So `javafx-maven-plugin-version` can simply become `javafx.plugin.version`.